packages/turf-point-grid/index.js

```diff
@@ -1,73 +1,72 @@
+    var units = options.units;
+    var maskIsPoly = options.mask && (getType(options.mask) === 'Polygon' || getType(options.mask) === 'MultiPolygon');
+    var properties = (options.properties || {});
 
     // Containers
     var results = [];
-    var bboxMask = bbox;
 
     // validation
+    if (cellSide === null || cellSide === undefined) throw new Error('cellSide is required');
+    if (!isNumber(cellSide)) throw new Error('cellSide is invalid');
     if (!bbox) throw new Error('bbox is required');
-    if (!Array.isArray(bbox)) bbox = turfBBox(bbox); // Convert GeoJSON to bbox
+    if (!Array.isArray(bbox)) throw new Error('bbox must be array');
     if (bbox.length !== 4) throw new Error('bbox must contain 4 numbers');
-    if (!isObject(options)) throw new Error('options is invalid');
 
-    var  west = bbox[0];
-    var  south = bbox[1];
-    var  east = bbox[2];
-    var  north = bbox[3];
+    var west = bbox[0];
+    var south = bbox[1];
+    var east = bbox[2];
+    var north = bbox[3];
 
-    var  xFraction = cellSide / (distance(point([west, south]), point([east, south]), options));
-    var  cellWidth = xFraction * (east - west);
-    var  yFraction = cellSide / (distance(point([west, south]), point([west, north]), options));
-    var  cellHeight = yFraction * (north - south);
+    var xFraction = cellSide / (distance(point([west, south]), point([east, south]), units));
+    var cellWidth = xFraction * (east - west);
+    var yFraction = cellSide / (distance(point([west, south]), point([west, north]), units));
+    var cellHeight = yFraction * (north - south);
 
-    var  bboxHorizontalSide = (east - west);
-    var  bboxVerticalSide = (north - south);
-    var  columns = Math.floor(bboxHorizontalSide / cellWidth);
-    var  rows = Math.floor(bboxVerticalSide / cellHeight);
+    var bboxWidth = (east - west);
+    var bboxHeight = (north - south);
+    var columns = Math.floor(bboxWidth / cellWidth);
+    var rows = Math.floor(bboxHeight / cellHeight);
     // adjust origin of the grid
-    var  deltaX = (bboxHorizontalSide - columns * cellWidth) / 2;
-    var  deltaY = (bboxVerticalSide - rows * cellHeight) / 2;
-
-    var  bboxIsPoly = !Array.isArray(bboxMask) && (getType(bboxMask) === 'Polygon' || getType(bboxMask) === 'MultiPolygon');
+    var deltaX = (bboxWidth - columns * cellWidth) / 2;
+    var deltaY = (bboxHeight - rows * cellHeight) / 2;
 
-    var  currentX = west + deltaX;
+    var currentX = west + deltaX;
     while (currentX <= east) {
-        var  currentY = south + deltaY;
+        var currentY = south + deltaY;
         while (currentY <= north) {
-            var  pt = point([currentX, currentY], properties);
-            if (bboxIsMask === true && bboxIsPoly) {
-                if (inside(pt, bboxMask)) {
+            var pt = point([currentX, currentY], properties);
+            if (maskIsPoly) {
+                if (inside(pt, options.mask)) {
```

Comment:
For all optional parameters, you should use the param name `mask` instead of `options.mask`.